app/components/Form/ImageUploadField.css

```diff
@@ -7,4 +9,12 @@
   width: 100%;
   height: 300px;
   background: var(--additive-background);
+
+  @media (--mobile-device) {
+    height: 200px;
+  }
+
+  @media (--small-viewport) {
+    height: 150px;
```

Comment:
Would it be better to just add an aspect-ratio?

Reply:
I believe an aspect-ratio of `3.3` matches our image dimensions pretty well

Reply:
Good catch - updated to use aspect-ratio now 🚀